models/Posts.js

```diff
@@ -391,6 +396,40 @@ PostSchema.statics.updateComment = function (postId, commentId, commentData) {
   });
 };
 
+PostSchema.statics.likeComment = function (postId, commentId, user, addLike) {
```

Comment:
I know you asked about this elsewhere, but if it were up to me I would make this a `method` instead of a `static`.

& then put in the controller
```
Post.get(pid).then((post) => {
  post.likeComment(cid).then((comment) => {
    ...
  })
})
```
or if we made comment it's own schema,
```
Post.get(pid).then((post) => {
  post.comments.id(cid).like().then((comment) => {
    ...
  })
})
```

Although others may have different opinions. I can see the case for moving the `find` logic outside of the controller, but to me liking a post is something done on a specific document, so makes sense to be a method as opposed to a static.

Reply:
I'll leave this for later then, when one of us rewrites the Post schema methods. If someone is motivated to do the front end side of it, I'll hurry up and fix this